keyboards/melgeek/mj61/rev2/rev2.c

```diff
@@ -0,0 +1,211 @@
+    {0, CS18_SW7, CS17_SW7, CS16_SW7}, /* RGB33 */
+    {0, CS18_SW8, CS17_SW8, CS16_SW8}, /* RGB34 */
+    {0, CS18_SW9, CS17_SW9, CS16_SW9}, /* RGB36 */
+    {0, CS30_SW1, CS29_SW1, CS28_SW1}, /* RGB64 */
+    {0, CS30_SW2, CS29_SW2, CS28_SW2}, /* RGB65 */
+    {0, CS30_SW3, CS29_SW3, CS28_SW3}, /* RGB66 */
+    {0, CS30_SW4, CS29_SW4, CS28_SW4}, /* RGB67 */
+    {0, CS30_SW5, CS29_SW5, CS28_SW5}, /* RGB68 */
+    {0, CS30_SW6, CS29_SW6, CS28_SW6}, /* RGB69 */
+    {0, CS30_SW7, CS29_SW7, CS28_SW7}, /* RGB70 */
+    {0, CS21_SW1, CS20_SW1, CS19_SW1}, /* RGB37 */
+    {0, CS21_SW2, CS20_SW2, CS19_SW2}, /* RGB40 */
+    {0, CS21_SW3, CS20_SW3, CS19_SW3}, /* RGB43 */
+    {0, CS21_SW4, CS20_SW4, CS19_SW4}, /* RGB38 */
+    {0, CS21_SW5, CS20_SW5, CS19_SW5}, /* RGB41 */
+    {0, CS21_SW6, CS20_SW6, CS19_SW6}, /* RGB44 */
+    {0, CS21_SW7, CS20_SW7, CS19_SW7}, /* RGB45 */
+    {0, CS21_SW8, CS20_SW8, CS19_SW8}, /* RGB39 */
+    {0, CS21_SW9, CS20_SW9, CS19_SW9}, /* RGB42 */
+    {0, CS30_SW8, CS29_SW8, CS28_SW8}, /* RGB71 */
+};
+
+led_config_t g_led_config = {
+	{
+		/* C0     C1         C2        C3        C4        C5        C6        C7        C8          C9        C10         C11        C12        C13  */
+		{  0,      1,         2,        3,        4,        5,        6,        7,        8,          9,        10,         11,       12,        14     }, /* R0 */
+		{ 16,     17,        18,       19,       20,       21,       22,       23,       24,         25,        26,         27,       28,        29     }, /* R1 */
+		{ 30,     31,        32,       33,       34,       35,       36,       37,       38,         39,        40,         41,       43,        NO_LED }, /* R2 */
+		{ 46,     NO_LED,    48,       49,       50,       51,       52,       53,       54,         55,        56,         57,       59,        NO_LED }, /* R3 */
+		{ 61,     62,        NO_LED,   63,       NO_LED,   65,       NO_LED,   NO_LED,   NO_LED,     67,        68,         69,       NO_LED,    70     }, /* R4 */
+    }, {
+        {0, 0},   {16, 0},   {32, 0},  {48, 0},  {64, 0},  {80, 0},  {96, 0},  {112, 0},  {128, 0},  {144, 0},  {160, 0},  {176, 0},  {192, 0},  {208, 0},  {224, 0},  {240, 0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32}, {192, 32}, {208, 32}, {224, 32},
+		{0, 48},  {16, 48},  {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48}, {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48}, {192, 48}, {208, 48}, {224, 48}, {240, 48},
+		{0, 64},  {16, 64},  {32, 64}, {48, 64}, {64, 64}, {80, 64}, {96, 64}, {112, 64}, {128, 64}, {144, 64},
+	}, {
+	       4,      4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,         4,         4,
```

Comment:
All alphas, no modifiers?

Reply:
in fact, I don't very clear about modifer flag of led

Reply:
The flags are used by a number of animations.  
https://docs.qmk.fm/#/feature_rgb_matrix?id=flags

So you can have alphas (keylight), modifiers (esc, tab, caps, shift, control, etc), underglow, and indicators.   

Additionally, people (like myself) use these flags for layer indication and the like.